package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "qb-schema",
+  "private": true,
+  "description": "API and schema documentation for describing quiz tournaments.",
+  "version": "1.4.1",
```

Comment:
This should just be `1.4`, right? Elsewhere we say 1.4.

Reply:
I'm tempted to change everything to 1.4.1; there's one subtle change where providing both `bonus_points` and `bonus` on a `MatchQuestion` has gone from discouraged to invalid.

(That said, for future versions, significant software changes even absent schema changes should probably represent a version bump of some sort.)